### PR TITLE
Add OIDC logout 

### PR DESCRIPTION
**Pull request by utle**
Add OIDC logout

### CODE REVIEW

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#run libbybot

**Comment by utle** [comment]
#run-libby-bot

File: dev/io.openliberty.security.oidcclientcore.internal/src/io/openliberty/security/oidcclientcore/logout/RPInitiatedLogoutStrategy.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        params.add(new BasicNameValuePair(ID_TOKEN_HINT, idToken));
+        params.add(new BasicNameValuePair(CLIENT_ID, clientId));
+        params.add(new BasicNameValuePair(POST_LOGOUT_REDIRECT_URI, redirectURI));
+
+        return new ProviderAuthenticationResult(AuthResult.REDIRECT_TO_PROVIDER, HttpServletResponse.SC_OK, null, null, null, endSessionEndPoint);
+
+    }
+
+    public String getIdToken() {
+        return req.getParameter(TokenConstants.ID_TOKEN);
```
Comment (teddyjtorres) [inline dev/io.openliberty.security.oidcclientcore.internal/src/io/openliberty/security/oidcclientcore/logout/RPInitiatedLogoutStrategy.java:70]: The id token will come from the internal context, passed as a parameter to this strategy class. We can deliver this and update in a follow up PR.

**Review by teddyjtorres — APPROVED** [review]

**Pull request merged**